src/load_routes.js

```diff
@@ -28,10 +28,8 @@ const defineRoutes = (prefix, key, schema, database, subscribers) => {
 // load route
 export default config => {
   const subscribers = (config.subscribers || []).map(s => s.implementation);
-
-  const routes = config.schema.map(({schema}) =>
-    _.keys(schema).map(key =>
-      defineRoutes(config.prefix, key, schema[key], config.database, subscribers))
+  const routes = _.keys(config.database.models).map(key =>
+    defineRoutes(config.prefix, key, config.database.models[key], config.database, subscribers)
```

Comment:
So I was surprised this worked at first, then traced the code and realized that we dont use the model (was schema) 

Please take out that 3rd parameter from the call and defineRoutes function

thanks,